cohttp-eio/examples/client_proxy.ml

```diff
@@ -48,8 +48,6 @@ let run_client url cacert all_proxy no_proxy http_proxy https_proxy proxy_auth =
       proxy_auth
   in
 
-  Mirage_crypto_rng_unix.use_default ();
```

Comment:
actually it was probably better defined close the Eio_main.run call. Sorry to nitpick on this...

Reply:
No worries. I but it right under the main now in a fixup, matching the position from 

https://github.com/mirage/ocaml-cohttp/blob/bb0f2c4de3f9aba1e1f6abd2c67e14a36bc7dc02/cohttp-eio/examples/client_tls.ml#L27-L31